app/main.cpp

```diff
@@ -54,26 +52,28 @@ void installTranslator( PencilApplication& app )
     qDebug() << "Install translation = " << b;
 }
 
-int handleArguments( MainWindow2 & mainWindow )
+int handleArguments( PencilApplication& app )
 {
+    QTextStream out( stdout );
+    QTextStream err( stderr );
     QStringList args = PencilApplication::arguments();
     QString inputPath;
     QStringList outputPaths;
     int width = -1, height = -1;
     bool transparency = false;
 
     QCommandLineParser parser;
-    // TODO: Ignore -NSDocumentRevisionsDebugMode
+    args.removeOne("-NSDocumentRevisionsDebugMode");
 
     parser.setApplicationDescription( PencilApplication::tr("Pencil2D is an animation/drawing software for Mac OS X, Windows, and Linux. It lets you create traditional hand-drawn animation (cartoon) using both bitmap and vector graphics.") );
     parser.addHelpOption();
     parser.addVersionOption();
     parser.addPositionalArgument( "input", PencilApplication::tr( "Path to the input pencil file." ) );
 
-    QCommandLineOption exportSeqOption( QStringList() << "o" << "export-sequence",
+    QCommandLineOption exportOutOption( QStringList() << "o" << "export-sequence" << "export-movie",
```

Comment:
Having --export-sequence and --export-movie do the same thing is a bit weird as it allows for things like: `--export-sequence test.mp4` or `--export-movie test.png`. We should either just use --export for the long form, or handle them separately.

Reply:
Fair point. I’ll make --export the primary long form and turn --export-sequence into a hidden option for bc.